app/CMakeLists.txt

```diff
@@ -189,7 +189,7 @@ endif()
 # DEMO: full physics loop
 #-----------------------------------------------------------------------------#
 
-if(CELERITAS_BUILD_DEMOS AND CELERITAS_USE_VecGeom)
+if(CELERITAS_BUILD_DEMOS AND CELERITAS_USE_VecGeom AND CELERITAS_USE_Geant4)
```

Comment:
Per the slack discussion -- the app itself will build fine without geant4, it's just the testing that requires it (and also ROOT) to generate and use the data. Can you revert this line, then add restrictions to the test itself?

Reply:
Done.